shared/triage_metadata.go

```diff
@@ -265,6 +265,23 @@ func (tm triageMetadata) Triage(metadata MetadataResults) error {
 	}
 
 	triagedMetadataMap := addToFiles(metadata, filesMap, tm.logger)
-	tm.metadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
+	tm.MetadataGithub.wptmetadataGitHubInfo = getWptmetadataGitHubInfo(tm.ctx, tm.githubClient)
 	return tm.createWPTMetadataPR(triagedMetadataMap)
 }
+
+// GetTriageMetadata returns an instance of triageMetadata struct.
```

Comment:
```suggestion
// GetTriageMetadata returns an instance of the triageMetadata struct.
```

(Though nit; as a public method, this comment could be more useful if it explained what the triage metadata struct is used for. Not a big worry though.)